index.html

```diff
@@ -822,6 +836,19 @@ <h2>Exposing change of MediaStreamTrack configuration</h2>
       </div>
     </p>
     </div>
+    <h3>Example</h3>
+    <div>
+    <p>This example shows how to monitor external background blur changes.</p>
+    <pre class="example">
+const stream = await navigator.mediaDevices.getUserMedia({ video: true });
+const [track] = stream.getVideoTracks();
+track.addEventListener("configurationchange", (event) => {
+  // Background blur configuration may have changed.
+  if (track.getSettings().backgroundBlur) {
+    // Turn off web app own blurring if needed so that video doesn't get double-blurred.
+  }
```

Comment:
Also, thanks for adding an example! Two nits (open to discuss):
1. style consistency with [existing examples](https://w3c.github.io/mediacapture-main/getusermedia.html#example-15): `{}` spacing, and `()` around single-arg arrow functions
2. Could we show how to detect change with code instead? E.g.
```suggestion
const stream = await navigator.mediaDevices.getUserMedia({video: true});
const [track] = stream.getVideoTracks();
let {backgroundBlur} = track.getSettings();

track.addEventListener("configurationchange", event => {
  if (backgroundBlur != track.getSettings().backgroundBlur) {
    // Turn on/off software blurring as needed so that video doesn't get double-blurred.
    backgroundBlur = track.getSettings().backgroundBlur;
  }
```

Reply:
Got it.